notebooks/official/automl/get_started_automl_training.ipynb

```diff
@@ -949,7 +915,7 @@
       "source": [
         "#### Undeploy the model\n",
         "\n",
-        "When you are done doing predictions, you undeploy the model from the `Endpoint` resouce. This deprovisions all compute resources and ends billing for the deployed model."
+        "When you're done doing predictions, you undeploy the model from the `Endpoint` resouce. This deprovisions all compute resources and ends billing for the deployed model."
```

Comment:
delete "you" and fix typo ("resouce"):
- change to: "When you're done doing predictions, undeploy the model from the `Endpoint` resource."